impl/src/testing.rs

```diff
@@ -45,6 +46,48 @@ dependencies = [
     "
 }
 
+pub fn advanced_toml() -> &'static str {
+  r#"
+[package]
+name = "cargo-raze-test"
+version = "0.1.0"
+
+[lib]
+path = "not_a_file.rs"
+
+[dependencies]
+proc-macro2 = "1.0.24"
+"#
+}
+
+pub fn advanced_lock() -> &'static str {
```

Comment:
I would use `indoc` to make the function / const indented. I would also consider a const here

Reply:
This was done to be consistent with what was there before (`basic_toml` and `basic_lock`) but I can update it.

Reply:
How would I apply `const` here? I thought that was implicit

Reply:
I've updated these now.

Reply:
```rust
pub const fn advanced_lock()
```

or 

```rust
const ADVANCED_LOCK: &str = indoc! { r#" str str str "#}
```